articles/openshift/oshift-how-view-proxy-logs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to view proxy logs in a multi-network OpenShift cluster | UKCloud Ltd
+description: Explains how to view the logs of the squid proxy in your cluster
+services: openshift
+author: Kieran O'Neill
+reviewer:
+lastreviewed: 03/12/2019
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: View proxy logs in a multi-network OpenShift cluster
+toc_fullpath: How To/oshift-how-view-proxy-logs.md
+toc_mdlink: oshift-how-view-proxy-logs.md
+---
+
+# How to view proxy logs in a multi-network OpenShift cluster
+
+## Overview
+
+UKCloud for OpenShift enables you to develop, deploy, and manage digital and container-based applications seamlessly across local physical or virtual environments, with full portability to and from UKCloud.
+
+This article explains how you can view the connection logs generated by the squid proxy in your OpenShift cluster to identify potential connection issues, such as attempting to access endpoints that aren't whitelisted.
+
+### Prerequisites
+
+Your OpenShift cluster must have access to multiple networks, be version 3.11 or later, and must have proxy logging enabled. Any OpenShift clusters deployed from November 2019 will have proxy logging enabled. If you have an older OpenShift cluster and would like proxy logging enabled, raise a request on the UKCloud Portal for this functionality.
```

Comment:
We should retrofit to all clusters - update this line to either remove the bit about raising a request or to state that we're doing it automatically.